### ISSUE TITLE
cuRL preview command is incorrect for Digest Auth. Quotes in header needs escaping 

### ISSUE THREAD

**Issue by dlancea** [a85/POSTMan-Chrome-Extension#1019]
When using Digest Auth, the header argument in Preview -> Curl looks like:

```
curl -X POST -H "Authorization: Digest username="test", realm="Test", nonce="45h6l34k5h6l34k56jh", uri="/test/", response="j6h345lkj6h34lj6h34l6kj", opaque="""
```

This fails due to not escaping the double quotes. It does work if the header argument is single quoted:

```
curl -X POST -H 'Authorization: Digest username="test", realm="Test", nonce="45h6l34k5h6l34k56jh", uri="/test/", response="j6h345lkj6h34lj6h34l6kj", opaque=""'
```

**Comment by abhijitkane** [comment]
We'll push a fix for this soon.

**Comment by a85** [comment]
We are moving to a more comprehensive code generator and this should be fixed with v3.0.
